### PR TITLE
Create stub ExecutionPayload for block in BlockProposalTestUtil

### PR DESCRIPTION
**Pull request by Nashatyrev**
## PR Description

- Create stub `ExecutionPayload` for block in `BlockProposalTestUtil`
- Fix `Fix ExecutionEngineClient.Stub` params order

### CODE REVIEW

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/ExecutionPayloadUtil.java [hunk 1]
```diff
@@ -15,7 +15,9 @@ public ExecutionPayloadUtil(ExecutionEngineService executionEngineService) {
     this.executionEngineService = executionEngineService;
   }
 
-  public ExecutionPayloadUtil() {}
+  public ExecutionPayloadUtil() {
+    this(ExecutionEngineService.createStub());
```
Comment (mkalinin) [inline ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/ExecutionPayloadUtil.java:19]: I was thinking about this way of doing things but decided that explicit NPE (if no engine service is set) would be better than implicit misbehaviour. Nevertheless, I am completely OK with this change if you need this behaviour somewhere.

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/client/ExecutionEngineClient.java [hunk 2]
```diff
@@ -47,8 +47,8 @@
           return SafeFuture.completedFuture(
               new Response<>(
                   new ExecutionPayload(
-                      request.parentHash,
                       Bytes32.random(),
+                      request.parentHash,
```
Comment (mkalinin) [inline ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/client/ExecutionEngineClient.java:51]: What a fix! 🥇

**Review by mkalinin — APPROVED** [review]

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/ExecutionPayloadUtil.java (hunk 1 above)
Comment (Nashatyrev) [reply ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/ExecutionPayloadUtil.java:19]: Yeah, let it be that way for now. I think that both cases would need to refactored in production. I mean that calling `setExecutionEngineService` later doesn't look too safe to me as well

**Comment by Nashatyrev** [comment]
Merge it plz, I have no write access to your repo

**Pull request merged**